packages/drafts-realm/user-email.gts

```diff
@@ -0,0 +1,127 @@
+import {
+  FieldDef,
+  primitive,
+  useIndexBasedKey,
+  deserialize,
+  BaseInstanceType,
+  BaseDefConstructor,
+  queryableValue,
+  field,
+  contains,
+  StringField,
+} from 'https://cardstack.com/base/card-api';
+import { Component } from 'https://cardstack.com/base/card-api';
+import { BoxelInput, FieldContainer } from '@cardstack/boxel-ui/components';
+import { TextInputValidator } from '../base/text-input-validator';
+
+function isValidEmail(email: string): boolean {
+  let emailRegex = new RegExp(
```

Comment:
I’ve never seen an email regex done right in over 30 years of experience 😄. Email format validation is actually a really hard thing to do. Perhaps we should just check for really simple things, like an “@“ and a “.” separating the host from the TLD.